### DISCUSSION TITLE
Can't get data from inverters while running off-grid

### DISCUSSION THREAD

**Discussion by HoriCoX24** [wlcrs/huawei_solar#134] [Setting up the integration]
Hello!

I am facing with a problem while my inverters are running in off-grid mode. 
Retrying setup: Could not update 102070083621 values:
The inverters are connected via the smartdongle. The ip is the same, i am able to ping them. Any idee what causes this? I will provide more data if needed. Thank you!

**Comment by wlcrs** [comment]
Issue #108 has led me to believe that the inverter aggressively closes the modbus connections in off grid mode. If you want me to review your case, then please properly fill in a bug report, including logs of pymodbus as described in the Readme/new bug report issue page.

**Reply by HoriCoX24** [reply]
Thank you for the quick reply! I will go and fill a bug report. Thank you!
